packages/jest-did-matcher/src/matchers/toBeDidCoreDatetime/predicate.js

```diff
@@ -1,5 +1,4 @@
 export default expected => {
-    const regex = /\d{4}-\d{2}-\d{2}T\d{2}:\d{2}:\d{2}Z/
+    const regex = /[0-9]{4}-[0-1][0-9]-[0-3][0-9]T[0-2][0-9]:[0-5][0-9]:[0-5][0-9]Z/
```

Comment:
I suggest we use the official definition for the XMLDateTime regex:

https://www.w3.org/TR/xmlschema11-2/#nt-dateTimeRep

This is not something we should be attempting to create on our own.

Reply:
I agree, but I don't know how to create an EBNF parser in JavaScript, or if some existing library can be used for parsing XML dateTime. So someone else would have to do that...

Reply:
> I agree, but I don't know how to create an EBNF parser in JavaScript

That specification has a regex that you can copy/paste into JavaScript... look further down the page.